lib/components/molecules/Form/Form.js

```diff
@@ -15,95 +16,67 @@ import FieldFileInput from './FieldFileInput';
 import DDFileInput from './DragDropInput';
 
 type Props = {
-  className?: string,
-  action: string,
   initialValues?: any,
   children: Node,
-  handleSubmit: Function,
+  onSubmit: Function,
   debug?: boolean,
   validationSchema?: Object | null,
-  validate?: Function | null,
-  validateOnChange?: boolean,
-  validateOnBlur?: boolean,
-  formWrapperProps?: Object,
+  validate?: String,
   yupValidationSchema?: Function | null,
 };
 
-class Form extends React.PureComponent<Props> {
-  // eslint-disable-next-line react/sort-comp
-  getSchema = (
-    yupValidationSchema: Function | null = null,
-    validationSchema: Object | null = null
-  ) => {
+const Form = ({
+  onSubmit,
+  initialValues,
+  children,
+  debug,
+  validate,
+  validationSchema,
+  yupValidationSchema,
+}: Props) => {
+  const getSchema = () => {
     if (yupValidationSchema) return yupValidationSchema;
     return validationSchema ? buildSchema(validationSchema) : null;
   };
+  const { handleSubmit, register, errors } = useForm({
+    mode: validate,
+    validationSchema: getSchema(yupValidationSchema, validationSchema),
```

Comment:
Do we still want to continue using Yup for validation? This was a dependency in formik may be we can avoid it. Thoughts?

Reply:
we can also do the validation using yup in react-hook-form